includes/data/user-investments.php

```diff
@@ -20,9 +20,10 @@ class WDGUserInvestments {
 	
 	public function __construct( $WDGInvestorEntity ) {
 		$this->wp_ref = $WDGInvestorEntity->get_wpref();
-		if ( WDGOrganization::is_user_organization( $WDGInvestorEntity->get_wpref() ) ) {
+		if ( get_class( $WDGInvestorEntity ) == 'WDGOrganization' ) {
```

Comment:
En fait, j'aurais laissé l'ancienne condition, mais t'as raison, y'a un truc différent qui peut être fait. Si c'est une orga, on peut construire un objet orga à partir du wpref, du style : 
$this->orga = new WDGOrganization( $this->wp_ref );

Reply:
done